Doc/library/ftplib.rst

```diff
@@ -74,9 +77,11 @@ The module defines the following items:
 
    .. versionchanged:: 3.9
       If the *timeout* parameter is set to be zero, it will raise a
-      :class:`ValueError` to prevent the creation of a non-blocking socket
+      :class:`ValueError` to prevent the creation of a non-blocking socket.
+      The *encoding* parameter was added, and the default was changed from
+      Latin-1 to UTF-8.
```

Comment:
I may be good to mention that the change was motivated by respecting :rfc:`2640`. It's always good to rely on standards :-) Same remark for following versionchanged (FTP TLS) and the NEWS entry.